README.md

```diff
@@ -196,6 +196,22 @@ Note that the default hyperparameters used in the zoo when tuning are not always
 
 When working with continuous actions, we recommend to enable [gSDE](https://arxiv.org/abs/2005.05719) by uncommenting lines in [utils/hyperparams_opt.py](https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/utils/hyperparams_opt.py).
 
+
+## Experiment tracking
+
+We support tracking experiment data such as learning curves and hyperparameters via [Weights and Biases](https://wandb.ai).
```

Comment:
maybe put a link to SB3 integration doc?

Reply:
The thing is we are not really using the SB3 WANDB integration. The reason is that `args` variable in `train.py` contains the hyper-parameter info and we kind of no longer need SB3 WANDB integration.

Reply:
ok

Reply:
> and https://wandb.ai/openrlbenchmark/sb3/runs/1b65ldmh is a 404 for me :/

This is fixed

Reply:
> a comment is missing then ;)

Added a comment.